SU-SWS/stanford_profile
# Added layout builder role

# READY FOR REVIEW

# Summary
- New "Layout Builder User" role to remove the access for normal users.
- Allows site managers the ability to upgrade themselves to use layout builder.
- For existing sites with custom layout builder content: https://github.com/SU-SWS/stanford_profile_helper/pull/142

# Need Review By (Date)
- 9/30

# Urgency
- medium

# Steps to Test
1. checkout this branch
2. `drush cim` or `blt drupal:update`
3. Log in or masquerade as a site manager
4. verify you don't get the "Layout" tab on content.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
